protocol/v2/ssv/runner/committee.go

```diff
@@ -240,15 +250,21 @@ func (cr *CommitteeRunner) ProcessConsensus(logger *zap.Logger, msg *spectypes.S
 				zap.String("signature", hex.EncodeToString(partialMsg.PartialSignature[:])),
 			)
 		case spectypes.BNRoleSyncCommittee:
+			validDuties++
 			blockRoot := beaconVote.BlockRoot
 			partialMsg, err := cr.BaseRunner.signBeaconObject(cr, duty, spectypes.SSZBytes(blockRoot[:]), duty.DutySlot(),
 				spectypes.DomainSyncCommittee)
 			if err != nil {
 				return errors.Wrap(err, "failed signing sync committee message")
 			}
 			postConsensusMsg.Messages = append(postConsensusMsg.Messages, partialMsg)
+		default:
+			return fmt.Errorf("invalid duty type: %s", duty.Type)
 		}
 	}
+	if validDuties == 0 {
+		return errors.New("no valid duties")
+	}
```

Comment:
Perhaps, can we set the Runner as "Finished" here as well?

Reply:
@MatheusFranco99 done